serverless/auto_segmentation/nuclio/mask_rcnn.py

```diff
@@ -0,0 +1,105 @@
+
+# Copyright (C) 2018-2019 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import numpy as np
+import sys
+import skimage.io
+from skimage.measure import find_contours, approximate_polygon
+
+class MASKRCNN:
+    def get_labels_by_name(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5729179)